corona-warn-app/cwa-app-android
# Show badge count and badge text under certificate QR (EXPOSUREAPP-9194)

How to test:

- Scan a few certificates of different types
- Change env to invalidate certs
- Make sure the count is displayed right and once all the badges are dismissed the invalid error message with the traingle is displayed instead
- From a blank slate, scan again a few certs and travel to the future, after the expiration date of the certificates and check that the badges show up and dissapear with the "Expired" text and icon taking their place.
- Scan a booster vacc cert #4007 and go through the process described in the mentioned PR
- Make sure the badge shows up for the certificate under the QR and once you enter the person details screen, nothing is shown under the QR code on the person overview screen.
